### PR TITLE
Add Yopass Extension

### PR DESCRIPTION
**Pull request by ricoberger**
## Description

Add an extension for [Yopass](https://github.com/jhaals/yopass). Yopass is a self hosted solution for sharing secrets securely.

## Screencast

yopass-1

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/yopass/src/encryptmessage.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    } catch (err) {
+      showToast({ style: Toast.Style.Failure, title: "An error occured", message: err.message });
+    }
+  };
+
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm title="Encrypt Message" onSubmit={submitEncryptMessage} />
```
Comment (pernielsentikaer) [inline extensions/yopass/src/encryptmessage.tsx:78]: Maybe include `Icon.Fingerprint` as icon?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @ricoberger 👋 

Thanks for your contribution, awesome extension! 🔥 

It could be nice with some [Validation](https://developers.raycast.com/api-reference/user-interface/form#validation) since it's possible to send the form without any content added 🙂

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/yopass/src/encryptmessage.tsx (hunk 1 above)
Comment (ricoberger) [reply extensions/yopass/src/encryptmessage.tsx]: 👍 done.

**Comment by ricoberger** [comment]
> It could be nice with some [Validation](https://developers.raycast.com/api-reference/user-interface/form#validation) since it's possible to send the form without any content added 🙂

Hi @pernielsentikaer thanks for your review 🙂. I added some validation for the message field, so that the field is required now.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Thanks for the update @ricoberger 🔥 

Looks good to me, approved 🔥

**Pull request merged**